cluster-api/src/main/java/io/aiven/klaw/clusterapi/services/KafkaConnectService.java

```diff
@@ -62,18 +60,13 @@ public Map<String, String> deleteConnector(ClusterConnectorRequest clusterConnec
               new ParameterizedTypeReference<>() {});
     } catch (RestClientException e) {
       log.error("Error in deleting connector ", e);
-      result.put("result", ApiResultStatus.ERROR.value);
-      result.put("errorText", e.toString().replaceAll("\"", ""));
-      return result;
+      return ApiResponse.builder().success(false).message(e.getMessage()).build();
```

Comment:
This is something I have been looking at and finding the e.getMessages() returning something like ->
"{"error_code":400,"message":"Connector name configuration (AindriuHundred) doesn't match connector name in the URL (fdsdddsa)"}"

Just wondering if you have seen this at all as I was basically going in and parsing out the message inside of the message and returning just that.

In your testing have you seen anything similar?